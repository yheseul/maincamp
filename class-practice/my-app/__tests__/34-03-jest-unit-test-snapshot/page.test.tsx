import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import JestUnitTestSnapshot from "@/app/section34/34-03-jest-unit-test-snapshot/page";

it("기존 사진이랑 바뀐 거 비교 - snapshot test", () => {
  const result = render(<JestUnitTestSnapshot />);
  expect(result.container).toMatchSnapshot();
});
