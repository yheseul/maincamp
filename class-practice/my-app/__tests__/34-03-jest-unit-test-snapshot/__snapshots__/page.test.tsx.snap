// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`기존 사진이랑 바뀐 거 비교 - snapshot test 1`] = `
<div>
  <div>
    <div>
      hello
    </div>
    hobby: 
    <input
      type="text"
    />
    <button>
      go
    </button>
  </div>
</div>
`;
